docs/feature-gates.md

```diff
@@ -310,3 +311,32 @@ Refer to this [document](service-loadbalancer.md) for more information.
 #### Requirements for this Feature
 
 This feature is currently only supported for Nodes running Linux.
+
+### ExternalNode
+
+The `ExternalNode` feature enables Antrea Agent runs on a Virtual Machine or a bare metal
+server which is not a Kubernetes Node.
+
+User creates ExternalNode object in the cluster to represent the Virtual Machine or the
+bare metal server, and then Antrea Controller creates an AntreaAgentInfo object for Agent
+that is running on the VM/BM. Antrea Controller also deletes the AntreaAgentInfo object when
+ExternalNode is deleted. Agent realizes the ExternalNode object on the VM/BM by attaching
+the interfaces on OVS, which is configured with the required IP and/or name.
+
+After user creates an ExternalNode object, Antrea Controller creates an ExternalEntity object
+for it, which has the same labels added on the ExternalNode object. User can apply an Antrea
+NetworkPolicy to the VM/BM by selecting the ExternalEntity object with these labels. The
+ExternalNode name is set on the `ExternalNode` field in the ExternalEntity. Antrea Controller
+uses this value to find the target Antrea Agent, send the Antrea NetworkPolicy internal
+objects, and then Antrea Agent running on the VM/BM realizes the corresponding rules.
+
+Antrea Agent supports the `ExternalNode` feature on both Linux and Windows.
+
+#### Requirements for this Feature
+
+Since it  means Antrea Agent is running on an unmanaged VM when this feature is enabled,
+other features designed for K8s Pods are disabled, e.g., AntreaIPAM, NodeIPAM, SecondaryNetwork.
+As of now, this feature requires that `AntreaProxy` and `NetworkPolicyStats` are also enabled.
+
+OVS is required to be installed on the Virtual Machine or the barematal server before running
```

Comment:
Virtual Machine

bare metal server (I think "bare-metal server" is better, but at least you should use a consistent format in all places).